R/create_soils.R

```diff
@@ -40,27 +41,34 @@ create_soils <- function(
     mustWork = FALSE
   )
 
-  if (fs::dir_exists(path)) {
-    if (!isTRUE(overwrite)) {
+  dir_exists <- fs::dir_exists(path)
+  overwriting <- isTRUE(overwrite)
+
+  if (dir_exists && !overwriting) {
       cli::cli_abort(
         c(
-          "!" = "{path} already exists.",
-          "i" = "Set `create_soils(overwrite = TRUE)` to overwrite anyway."
+          "!" = "{.path {path}} already exists.",
+          "i" = "To always overwrite: \\
+          {.code create_soils({.path {path}} overwrite = TRUE)}"
```

Comment:
```suggestion
          {.code create_soils({.str {path}} overwrite = TRUE)}"
```
to generate valid code, you may be better off using .str markup as it will quote the path.

Reply:
Good point! `.path` is neat, but in this use it generates a clickable link to the path, which doesn't really make sense:

<img width="402" alt="image" src="https://github.com/WA-Department-of-Agriculture/soils/assets/6488485/d4863493-1419-4a21-8bc5-429cc6ac6c4c">